Formula/abcmidi.rb

```diff
@@ -17,6 +17,9 @@ def install
     # be overridden and will be chosen over the "Makefile" file.
     rm "makefile"
 
+    # Fix "Failed to execute: ./configure" issue
+    chmod 0755, "./configure"
```

Comment:
is there anyway that we can send a patch to the upstream source code too?

I think I saw this similar issue before.

Reply:
I checked upstream (https://github.com/sshlien/abcmidi) and it appears to be fixed already, just not in this release